app/models/ship.rb

```diff
@@ -3,6 +3,10 @@ class Ship < ApplicationRecord
   has_many :bookings, dependent: :destroy
   has_many :users, :through => :bookings
 
+  validates :category, inclusion: { in: %w(Bateau Radeau Yacht Militaire Sous-marin Porte-Container Oil-tanker Cruise Small-boat Autres) }
```

Comment:
mettre plutot cette liste dans une constante pour pouvoir la rappeler dans les options du select :
CATEGORIES = %w(Bateau Radeau Yacht Militaire Sous-marin Porte-Container Oil-tanker Cruise Small-boat Autres)
   validates :category, inclusion: { in: CATEGORIES }